tox.ini

```diff
@@ -9,6 +9,10 @@ commands = pytest {posargs}
 [testenv:ci]
 deps = -r requirements/tests.txt
 commands = pytest -vv -m "not django_db"
+passenv = TEST_ORACLE_HOST TEST_ORACLE_PORT TEST_ORACLE_DB TEST_ORACLE_LOGIN TEST_ORACLE_PASSWORD \
```

Comment:
can we tell tox to pass all env variables instead?

Reply:
With `*` we could, is it what we want?

Reply:
I'd say yes

Reply:
don't know if it's better, I think I prefer `*` (less maintenance) but I'll let you judge